testutil/checkers_test.go

```diff
@@ -1,4 +1,5 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
+// +build !ppc
```

Comment:
Can we have a comment explaining why this is being excluded?  It doesn't feel reasonable upfront.